internal/app/go-filecoin/node/test/setup.go

```diff
@@ -0,0 +1,117 @@
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/go-filecoin/build/project"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-filecoin/internal/app/go-filecoin/node"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/clock"
+	gengen "github.com/filecoin-project/go-filecoin/tools/gengen/util"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/stretchr/testify/require"
+)
+
+func MustCreateNodesWithBootstrap(ctx context.Context, t *testing.T, additionalNodes uint) ([]*node.Node, context.CancelFunc) {
+	ctx, cancel := context.WithCancel(ctx)
+	nodes := make([]*node.Node, 1+additionalNodes)
+
+	// set up paths and fake clock.
+	presealPath := project.Root("fixtures/genesis-sectors")
+	genCfgPath := project.Root("fixtures/setup.json")
+	genTime := int64(1000000000)
+	blockTime := 1 * time.Second
```

Comment:
I'd prefer this to be 30s so it's clear that time is fake. I would guess we will later need to return the fake clock from this, for external manipulation.